source/contributing/howtogit.rst

```diff
@@ -0,0 +1,336 @@
+===============
+How to Git(Hub)
+===============
+
+If you want to assist in creating Sponge, you have an awesome addition to the API, or you want to improve our Docs,
+then you'll need to become familiar with ``git`` and GitHub. If you're already familiar with forking, branches,
+issues, pull-requests and commits, then just skip this topic. If you have no clue what we're talking about, then read on.
+
+.. note::
+  This guide assumes that you've read :doc:`../preparing/git` and that you've already setup your machine with a Git
+  client of your choice.
+
+The Basic Concept of Git and GitHub
+===================================
+
+Git allows many different developers to develop a single piece of software at the same time. GitHub is a website where
+developers can collaborate and share their work with others. GitHub relies on Git for the management of said work.
+
+.. note::
+  If you're unfamiliar with the Git and GitHub vocabulary, have a look at the
+  `glossary page on Github <https://help.github.com/articles/github-glossary/>`_.
+
+.. image:: /images/contributing/repo-overview.svg
+    :alt: Repo Overview
+
+In this case the repo is named ``SpongePowered``, has two branches named ``master`` and
+``feature 1`` and also some commits on both branches.
+
+Let's put these terms into context - beginning with the *repository*. The repository (short: *repo*) is the place where
+a project stores its files. The SpongePowered repositories are located at `GitHub <http://github.com/spongepowered>`__.
+However, this repo has some access restrictions to preserve it from unwanted or malicious changes. You can't simply make
+changes yourself, as the repo is read-only for regular users. Now you may wonder how you're supposed to file proposals
+and changes. Well, that's where *forks* come into play. You can grab a copy of the SpongePowered repos, and make your
+changes there. When you're done, you open it as a pull request (*PR*) on our repository. Your proposed additions and
+changes can then be reviewed, and staff will tell you if something is wrong, or needs improvement, and eventually merge
+the final PR.
+
+Here's a short summary of the procedure described above, before we go into detail:
+
+1. Fork the repo of your choice
+#. Clone it to your local machine
+#. Create a new branch
+#. Make the desired changes
+#. Test if everything works
+#. Commit the changes
+#. Sync them to GitHub
+#. Propose the changes in a PR to the SpongePowered Repo
+#. Amend to your PR if necessary
+#. Your PR gets pulled into master by staff
+
+Spare Me the Details!
```

Comment:
doesn't this literally mean the opposite of what you're trying to say? Maybe "Give Me the Full Details!" would be better.